<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简易聊天程序&lt;客户端&gt;</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/MainWindow/Resources/images/Smile-256.png</normaloff>:/MainWindow/Resources/images/Smile-256.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>交互窗口</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTextBrowser" name="display_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="message_Edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="send_bt">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/MainWindow/Resources/images/mail-reply-sender.png</normaloff>:/MainWindow/Resources/images/mail-reply-sender.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>网络</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>服务器地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IP_addr_Edit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入IP地址</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>端      口</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Port_Edit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>端口号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect_bt">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  连接  </string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/MainWindow/Resources/images/player_play.png</normaloff>:/MainWindow/Resources/images/player_play.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="un_connect_bt">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>断开连接</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/MainWindow/Resources/images/player_record.png</normaloff>:/MainWindow/Resources/images/player_record.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>337</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>(。・∀・)ノ</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>on_un_connect_bt_clicked()</signal>
 </slots>
</ui>
